sotodlib/tod_ops/fft_ops.py

```diff
@@ -180,13 +236,14 @@ def find_inferior_integer(target, primes=[2, 3, 5, 7, 11, 13]):
     while n > 0:
         n -= 1
         base = p**n
-        best_friend = find_inferior_integer(target / base, primes[1:])
+        best_friend = getattr(find_inferior_integer, "__wrapped__", find_inferior_integer)(target / base, primes[1:])
```

Comment:
Why not just `find_inferior_integer.__wrapped__`?

Reply:
Paranoia that someone may want to disable the cache by doing

`find_inferior_integer = find_inferior_integer.__wrapped__`

in a script.